redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -105,6 +122,9 @@
         return $list->getColumnLink('deleteType', '<i class="rex-icon rex-icon-delete"></i> ' . rex_i18n::msg('media_manager_type_delete'));
     });
 
+    $list->addColumn('copyType', '<i class="rex-icon rex-icon-copy"></i> ' . rex_i18n::msg('media_manager_type_copy'), -1, ['', '<td class="rex-table-action">###VALUE###</td>']);
+    $list->setColumnParams('copyType', ['func' => 'copy', 'type_id' => '###id###']);
```

Comment:
Ja, habe ich mir auch gedacht, deshalb ist es ja vorbereitet mit rex-icon-copy... aber es gibt aktuell noch nirgends eine solche Kopierfunktion oder? In dem Falle sollte man ein Icon raussuchen und dann rex-icon-copy benennen. Das hätte aber an sich ja nichts mit dem PR zu tun.